app/api/v2/handlers/agent_api.py

```diff
@@ -59,26 +59,32 @@ async def update_agent(self, request: web.Request):
         agent = await self.update_object(request)
         return web.json_response(agent.display)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'], summary="Update an agent, or if a existing agent match cannot be found, "
+                                                   "create one. Use the paw field in the URL to specify matching "
+                                                   "criteria and the fields from the AgentSchema in the request body"
+                                                   " to specify new field values.")
     @aiohttp_apispec.request_schema(AgentSchema(partial=True))
     @aiohttp_apispec.response_schema(AgentSchema)
     async def create_or_update_agent(self, request: web.Request):
         agent = await self.create_or_update_object(request)
         return web.json_response(agent.display)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'], summary="Delete an agent. Use the paw field in the URL to specify matching "
+                                                   "criteria for the agent(s) to delete.")
     @aiohttp_apispec.response_schema(AgentSchema)
```

Comment:
Can you add a `description` to the `@aiohttp_apispec.response_schema` for each of these endpoints?

Reply:
Yep, will do.

Reply:
I think this may have accidentally been left out of the last commit

Reply:
Yep, my bad. Should be fixed now.